backend/apps/owasp/admin.py

```diff
@@ -59,6 +60,13 @@ class EventAdmin(admin.ModelAdmin):
     search_fields = ("name",)
 
 
+class SnapshotAdmin(admin.ModelAdmin):
+    list_display = ("start_at", "end_at", "status", "created_at", "updated_at")
+    list_filter = ("status", "start_at", "end_at")
+    search_fields = ("status", "error_message")
+    ordering = ("-start_at",)
+
+
```

Comment:
Could you keep the formatting/style consistent with the current one? I mean the new class position within other classes and tuple formatting.